src/components/tabs/_tabs.scss

```diff
@@ -340,7 +345,10 @@
       }
     }
     @include max-breakpoint(bp--sm--major) {
-      height: rem(34px);
+      height: rem(40px);
+      &:first-of-type {
+        margin-top: rem(1px);
```

Comment:
Just wondered if we want spacing scale like `$spacing-4xs`.

Reply:
Good question. I hadn't seen a `$spacing-4xs` token but I can add it. Thanks @asudoh